src/data/uma-nickname.csv

```diff
@@ -142,13 +142,13 @@
 "真・女王の精神","True・Queen's Spirit"
 "時代の寵児","Era's Favorite"
 "勝利の探究者","Victory Seeker"
-"切磋琢磨",""
-"チームの繋がり",""
-"伝説の証人",""
-"チームの絆",""
+"切磋琢磨","Diligent Application"
+"チームの繋がり","Team Connection"
+"伝説の証人","Witness to a Legend"
+"チームの絆","Team Bonds"
 "逆転のウマ娘","Reversal Girl"
 "影をも恐れぬ怪物","Monster Who Fears No Shadow"
-"個人でもチームでも",""
-"アオハル魂爆発",""
+"個人でもチームでも","Individually and As a Team."
+"アオハル魂爆発","Aoharu soul explosion"
```

Comment:
As this is a title, i would title case it. That is to say, "Aoharu Spirit Explosion".
(Cygames officially calls these "Spirit" and not "Soul" explosions in their pages)

Reply:
Fixed in all files.